File: hello.erl
% Erlang

% This is Hello World
% written in Erlang.

-module(hello).
-export([start/0]).

hello(0) -> [];
hello(I) when I > 0 ->
  io:fwrite("Hello, World!\n"), % Greetings.
  hello(I-1).

start() -> hello(10).

File: hello.pl
# Perl

=begin comment
This is Hello World
written in Perl.
=end comment
=cut

sub hello {
  my $n = @_[0];
  for($i = 0; $i < $n; $i++) {
    print "Hello, World!\n" # Greetings.
  }
}

hello 10